src/rascal/models/kernels.hh

```diff
@@ -376,8 +376,10 @@ namespace rascal {
         return kernel->template compute<Property_t, Type>(
             managers_a, managers_b, representation_name);
       } else {
-        throw std::logic_error(
-            "The desired combination of parameters can not be handled.");
+        throw std::logic_error("Given kernel_type " +
+                            this->parameters["kernel_type"].get<std::string>() +
+                              " is not known."
+                              " It is 'Cosine'");
```

Comment:
I would have said `It must be one of: 'Cosine'`. But it's merged already, so keep that in mind for next time...